app/src/test/java/pm/gnosis/heimdall/ui/safe/details/transactions/AdapterEntryHeaders.kt

```diff
@@ -0,0 +1,7 @@
+package pm.gnosis.heimdall.ui.safe.details.transactions
+
+const val PENDING = "pending"
```

Comment:
What are these used for? Are they only for testing?

Reply:
Yes

Reply:
I would move them to the test folder in this case, else it looks good

Reply:
Ok